videoprocess/vpphdr_tm.cpp

```diff
@@ -296,7 +295,8 @@ video_frame_process(VASurfaceID in_surface_id,
     }
 
     hdrtm_filter_init(&filter_param_buf_id, g_tm_type);
-    hdrtm_metadata_init(out_metadata, g_tm_type);
+    VAHdrMetaDataHDR10 *out_hdr10_metadata = (VAHdrMetaDataHDR10*)malloc(sizeof(VAHdrMetaDataHDR10));
```

Comment:
Did not free it?

Reply:
Maybe we can just use stack var here. VAHdrMetaDataHDR10 out_hdr10_metadata = {} and pass its address to hdrtm_metadata_init function

Reply:
Or just using  static var  out_hdr10_metadata may also work for this cpp main file.

Reply:
> Maybe we can just use stack var here. VAHdrMetaDataHDR10 out_hdr10_metadata = {} and pass its address to hdrtm_metadata_init function

agree